src/sql/workbench/common/sqlWorkbenchUtils.ts

```diff
@@ -19,3 +19,41 @@ export function getSqlConfigValue<T>(workspaceConfigService: IConfigurationServi
 	let config = workspaceConfigService.getValue<{ [key: string]: any }>(ConnectionConstants.sqlConfigSectionName);
 	return config ? config[configName] : undefined;
 }
+
+/**
+ * Wraps provided string using Unicode character U+000A LINE FEED (LF) that qualifies as line break to wrap text in title attributes (tooltips).
+ * @param str string to be wrapped
+ * @param maxWidth max width to be allowed for wrapped text
+ * @returns wrapped string
+ */
+export function wrapStringWithUnicodeNewLine(str: string, maxWidth: number): string {
+	if (!str) {
+		return null;
+	}
+	let newLineStr = `
```

Comment:
I'd just remove this and use `'\n'` directly - easier to read and results in the same thing

Reply:
Thanks, just tested and it works!